guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc

```diff
@@ -4,7 +4,7 @@
 There are three main methods for registering hosts to {ProjectServer} or {SmartProxyServer}:
 
 . *(Default method)* Generate a `curl` command from {Project} and run this command from unlimited number of hosts to register them using the global registration template.
-This method is suited for both freshly installed hosts and hosts that have been previously registered, for example, to {Project} 5 or another {ProjectX}.
+This method is suited for both freshly installed hosts and hosts that have been previously registered, for example, to {Project} 5 or another {Project}.
```

Comment:
Same as above.

Reply:
Isn't this the very same place which you commented above? I only found this sentence twice in this file.